src/tech/v3/libs/smile/sparse_svm.clj

```diff
@@ -19,10 +19,12 @@
    The column of name `(options :sparse-column)` of `feature-ds` needs to contain the text as SparseArrays
    over the vocabulary."
   (let [train-array (into-array SparseArray (get feature-ds (options :sparse-column)))
-        score (get target-ds (first (ds-mod/inference-target-column-names target-ds)))]
+        score (get target-ds (first (ds-mod/inference-target-column-names target-ds)))
+        p (count (-> feature-ds meta :count-vectorize-vocabulary :vocab->index-map))
+        ]
```

Comment:
p can be calculated from teh vocabulary, no need to pass it in as option

Reply:
Is there a performance cost for calculating p?

Reply:
I ont' think so. It the size of a map.